### PR TITLE
Features/movie page api data

### PR DESCRIPTION
**Pull request by WojciechBylica**
the first step of APi data providing.
additional correction of exports in some components

### CODE REVIEW

**Review by KarolCiesluk — APPROVED** [review]
Great job Wojtek! 👍

**Review by patrycja-i-lesniak — APPROVED** [review]
The code looks fine at the moment.

File: src/features/MovieDetails/index.js [hunk 1]
```diff
@@ -3,25 +3,35 @@ import { Wrapper } from "./styled";
...
+                            <BigMovieTile movieData={APIData.movieData.data} />
+                            <PeopleTiles title="cast" creditsData={APIData.creditsData.data} />
+                            <PeopleTiles title="crew" creditsData={APIData.creditsData.data} />
+                        </Wrapper>
+                    </>);
+            case "error":
+                return (<Error />);
+            default:
+                return <NoResults />;
+        };
```
[the remark this answers is not in the document]
Comment (WojciechBylica) [reply src/features/MovieDetails/index.js:31]: corrected

File: src/features/MovieDetails/useAPIData.js [hunk 2]
```diff
@@ -12,19 +12,24 @@ export const useAPIData = () => {
...
+
+                const creditsAPI = `${APIAdress}movie/${movieId}/credits?${APIKey}`;
+                const response2 = await axios.get(creditsAPI);
+
                 setMovieData({
                     status: "success",
-                    movieData: response
+                    movieData: response,
+                    creditsData: response2
                 });
```
[the remark this answers is not in the document]
Comment (WojciechBylica) [reply src/features/MovieDetails/useAPIData.js:27]: done

File: src/features/MovieDetails/useAPIData.js [hunk 3]
```diff
@@ -12,19 +12,24 @@ export const useAPIData = () => {
             try {
                 const APIKey = 'api_key=3af561a12389e6d632bf79207cb88b6c';
                 const APIAdress = 'https://api.themoviedb.org/3/';
-                const movieId = '20';
-                const APIConfigurarion = `https://api.themoviedb.org/3/configuration?${APIKey}`;
-                // const  movieKeyWordsAPI = `https://api.themoviedb.org/3/movie/18/keywords?api_key=${APIKey}`;
+                const movieId = '21';
```
[the remark this answers is not in the document]
Comment (WojciechBylica) [reply src/features/MovieDetails/useAPIData.js:15]: lets use ""

**Comment by WojciechBylica** [comment]
added all needed API requests
movied useAPIData.js to src
updated PersonList to show the results of API data working
temporary moved MovieDetails to route path=Movies to check working of navigation

**Pull request merged**